README.md

```diff
@@ -10,21 +10,46 @@ For more information, visit us at [PHP Hampshire](http://phphants.co.uk).
 
 ### Prerequisites
 
- * PostgreSQL
- * Apache
- * PHP 7.1
+ * Docker
+ * Docker Compose
 
-### PHP Dependencies
+### Project setup
+
+First, you need to configure the application:
 
 ```bash
-git clone https://github.com/phphants/phph-site.git
-cd phph-site
-composer install
 cp config/autoload/local.php.dist config/autoload/local.php
-# Edit the local.php config to suit your environment
-vendor/bin/doctrine-migrations migrate
 ```
 
+Edit the configuration file accordingly with API tokens.
```

Comment:
Probably worth adding here that the postgres connection string needs putting into the config file too?